src/M6Web/Bundle/LogBridgeBundle/Matcher/Status/Type/CompleteType.php

```diff
@@ -26,7 +26,7 @@ protected function getPattern()
      */
     protected function transform($config)
     {
-        $refStatus = substr($config, 0, strlen($config - 2));
+        $refStatus = substr($config, 0, strlen((int) $config - 2));
```

Comment:
no
if we take $config = "4*" we have in your case
strlen($config) ==> 2
2-2 ==> 0
substr($config, 0, 0)

or we want
strlen($config - 2) ==> strlen(4 - 2) ==> strlen(2) ==> 1
substr($config, 0, 1)